docs/detections/building-block-rule.asciidoc

```diff
@@ -6,7 +6,7 @@ Create building block rules when you do not want to see their generated alerts
 in the UI. This is useful when you want:
 
 * A record of low-risk alerts without producing noise in the Alerts table.
-* Rules that execute on the alert indices (`.siem-signals-<kibana space>-*`).
+* Rules that execute on the alert indices (`.alerts-security.alerts-<kibana space>-*`).
```

Comment:
```suggestion
* Rules that execute on the alert indices (`.alerts-security.alerts-<kibana space>`).
```
We want to avoid having `-*` after the Kibana space name. We may want to consider updating the 7.x docs to remove the trailing `-*` as well, as it's not a great pattern to recommend to our users.

Reply:
Note that it's ok to have `-*` in place of the space name for cases where we want to cover all spaces - e.g. `.alerts-security.alerts-*`. This could be useful for specifying global admin privileges or the like.

Reply:
Excellent suggestions, thank you @marshallmain!